js/utils/object.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @param {any} obj
+ * @returns {boolean}
+ */
+export function is_object(obj) {
```

Comment:
This doesn't seem overly necessary, and is also exported despite being used once - in the function directly below.